cmd/internal/cli/apptainer.go

```diff
@@ -845,7 +845,7 @@ func getLibraryClientConfig(uri string) (*libClient.Config, error) {
 		if endpoint.DefaultLibraryURI != "" {
 			sylog.Warningf("no default remote in use, falling back to default library: %s", endpoint.DefaultLibraryURI)
 		} else {
-			return nil, fmt.Errorf("no default remote with library client in use")
+			return nil, fmt.Errorf("no default remote with library client in use (see \"https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote\")")
```

Comment:
Looks good, but I'm putting in a similar message in another PR and think it looks better without the double-quotes around the URL.  So please change it one more time to remove those escaped double-quotes.

Reply:
no problem! it's done